lib/timers.js

```diff
@@ -502,24 +502,26 @@ Timeout.prototype.close = function() {
 };
 
 
-var immediateQueue = {};
-L.init(immediateQueue);
+var immediateQueue = L.create();
```

Comment:
I bet the object ends up being access-optimized after enough runs anyway. Not to mention we _can_ make objects access-optimized explicitly. 

That said - this change improves style anyway and is better coding.